.env.example

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_BUILD_DATE = 20210917
```

Comment:
.env needs to be added to git ignore

Reply:
next uses .env.local

Reply:
My apologies, I didn't realise this was an example